api/v1beta2/cluster_types.go

```diff
@@ -585,6 +585,15 @@ type Topology struct {
 	// +listMapKey=name
 	// +kubebuilder:validation:MaxItems=1000
 	Variables []ClusterVariable `json:"variables,omitempty"`
+
+	// nodeDeletionStrategy specifies the strategy to delete nodes in the cluster.
+	// Available options:
+	// - graceful
+	// - force
```

Comment:
These should probably be PascalCase to align with other enum based APIs across kube